receiver/hazelcast.yaml

```diff
@@ -0,0 +1,5 @@
+hazelcast:
```

Comment:
The same question as with other guides, do we need this `hazelcast.yaml` at all?

Reply:
Removed it for the receiver, but for the publisher, we still need this file as we have `wan-replication` configs in it